src/lib/Behat/PageElement/Notification.php

```diff
@@ -48,6 +48,9 @@ public function getMessage(): string
     public function closeAlert(): void
     {
         $this->context->findElement($this->fields['closeAlert'])->click();
+        $this->context->waitUntil($this->defaultTimeout, function () {
+            return !$this->isVisible();
```

Comment:
`checkVisibilityByClass` has default timeout inside, equal to 5 seconds, so from the moment when Selenium noticed that element is not visible, we always have to wait 5s to get the `false` output.